README.md

```diff
@@ -171,6 +171,18 @@ interval between any two periodic sync events can be greater than the requested
 is up to the browser implemetation. Also, the interval between any two periodic sync events for a
 registration doesn't have to be the same.
 
+Since a website will most likely download resources from the network when processing a
+`periodicsync` event, multiple `periodicsync` events can cause a lot of data consumption. Like other
+service worker events, `periodicsync` MAY be subject to the service worker execution timeout. Every
+time the browser dispatches these events, the browser will potentially be kept awake for this
+duration, which is a few minutes. This can cause battery drain. The browser implementation SHOULD
```

Comment:
"which is a few minutes" --> which _could_ be a few minutes

Reply:
done